apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.sql

```diff
@@ -69,6 +69,6 @@ FROM
 WHERE
   g.id IS NULL
   AND e.course_instance_id = $course_instance_id
-  AND NOT users_is_instructor_in_course (e.user_id, e.course_instance_id)
```

Comment:
This was a bug: the second argument to this sproc should have been a course ID. I changed it to use the sproc that checks the course instance, which I think is what we want now.